src/plugins/tcl/CMakeLists.txt

```diff
@@ -2,13 +2,13 @@ if (DEPENDENCY_PHASE)
 	find_package (Boost)
 	if (NOT Boost_FOUND)
 		remove_plugin (tcl "boost not found, but needed for tcl plugin")
+	else ()
+		include_directories(SYSTEM ${Boost_INCLUDE_DIR})
```

Comment:
Can you remove this commit? Your solution includes the directory globally, target_include_directory is not yet available in cmake 2.8.8, thus I made a monstrous workaround in 97654f5e293f5c0de61041f3febc3ad5ebdbb1a6.